File: CONEXIONES/JAVA/MYSQLconex.java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

public class MYSQLconex {
    public static void main(String[] args) {
        String url = "jdbc:mysql://localhost:3306/"; 
        String usuario = "root"; 
        String contraseña = ""; 

        try {
            Connection conn = DriverManager.getConnection(url, usuario, contraseña);

            System.out.println("Conexión exitosa: " + conn);

            conn.close();
        } catch (SQLException e) {
            System.out.println("Error al conectarse a MySQL: " + e.getMessage());
        }
    }
}

File: CONEXIONES/JAVASCRIPT/MYSQLconex.js
const mysql = require('mysql');

const connection = mysql.createConnection({
    host: 'localhost',
    user: 'root',
    password: ''
});

connection.connect((err) => {
    if (err) {
        console.error('Error de conexión: ' + err.stack);
        return;
    }
    console.log('Conexión exitosa');
});

connection.end();